specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -7,6 +7,54 @@
   },
   "paths": {},
   "definitions": {
+    "AcsIncomingCallEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for an Microsoft.Communication.IncomingCall event",
+      "type": "object",
+      "properties": {
+        "to": {
+          "description": "The communication identifier of the target user.",
+          "$ref": "../../../../../communication/data-plane/Common/stable/2022-07-13/common.json#/definitions/CommunicationIdentifierModel"
+        },
+        "from": {
+          "description": "The communication identifier of the user who initiated the call.",
+          "$ref": "../../../../../communication/data-plane/Common/stable/2022-07-13/common.json#/definitions/CommunicationIdentifierModel"
+        },
+        "serverCallId": {
+          "description": "The Id of the server call",
+          "type": "string"
+        },
+        "callerDisplayName": {
+          "description": "Display name of caller.",
+          "type": "string"
+        },
+        "customContext": {
+          "description": "Custom Context of Incoming Call",
+          "type": "object",
+          "properties": {
+            "customHeaders": {
+              "type": "string",
+              "description": "Custom Headers for incoming call"
+            },
+            "sipHeaders": {
+              "type": "object",
+              "description": "Sip Headers for incoming call"
+            },
+            "voipHeaders": {
+              "type": "object",
+              "description": "Voip Headers for incoming call"
+            }
+          }
+        },
+        "incomingCallContext": {
+          "description": "Signed incoming call context.",
+          "type": "string"
+        },
+        "correlationId": {
```

Comment:
Is this a Guid? Is it input to any other client library APIs?

Reply:
not a guid